tests/daily_scraper/test_daily_scraper.py

```diff
@@ -6,309 +6,133 @@
-    'url': 'firefox/nightly/2013/09/2013-09-30-03-02-04-mozilla-central/firefox-27.0a1.en-US.win32.installer-stub.exe'
-    },
-
-    # -p win32 --branch=mozilla-central --stub (new format)
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-central',
-              'is_stub_installer': True},
-    'filename': '2013-10-01-03-02-04-mozilla-central-Firefox Installer.en-US.exe',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/Firefox Installer.en-US.exe'
-    },
-
-    # -p win32 --branch=mozilla-aurora
-    {'args': {'platform': 'win32',
-              'branch': 'mozilla-aurora'},
-    'filename': '2013-10-01-03-02-04-mozilla-aurora-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-aurora/firefox-27.0a1.en-US.win32.installer.exe'
-    },
-
-    # -p win32 --branch=ux
-    {'args': {'platform': 'win32',
-              'branch': 'ux'},
-    'filename': '2013-10-01-03-02-04-ux-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-ux/firefox-27.0a1.en-US.win32.installer.exe'
-    },
-
-    # --revision 1dbe350b57b1
-    {'args': {'platform': 'win32'},
-    'filename': '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
-    'url': 'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'
-    },
+    ({'platform': 'win32'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'),
+    ({'platform': 'win32', 'branch': 'mozilla-central'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.win32.installer.exe',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win32.installer.exe'),
+    ({'platform': 'win64', 'branch': 'mozilla-central'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.win64.installer.exe',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.win64.installer.exe'),
+    ({'platform': 'linux', 'branch': 'mozilla-central'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.linux-i686.tar.bz2',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.linux-i686.tar.bz2'),
+    ({'platform': 'linux64', 'branch': 'mozilla-central'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.linux-x86_64.tar.bz2',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.linux-x86_64.tar.bz2'),
+    ({'platform': 'mac', 'branch': 'mozilla-central'},
+     '2013-10-01-03-02-04-mozilla-central-firefox-27.0a1.en-US.mac.dmg',
+     'firefox/nightly/2013/10/2013-10-01-03-02-04-mozilla-central/firefox-27.0a1.en-US.mac.dmg'),
+    # Old stub format
```

Comment:
The specific test here isn't for an installer but for a txt file. So this comment doesn't apply. Please remove it and we seem to be able to land your changes. Thanks!